zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -206,9 +208,9 @@ public Factory(WebClient httpClient) {
     try (SafeCloseable ignored =
            Clients.withContextCustomizer(ctx -> ctx.logBuilder().name(name))) {
       HttpRequestWriter httpRequest = HttpRequest.streaming(request.headers());
-      response = httpClient.execute(httpRequest);
       request.writeBody(httpRequest::tryWrite);
```

Comment:
I noticed we aren't defensive here like we are on line 109 about empty content. should we be?

Reply:
In order to do that, I have to add a method or change the `writeBody` method in `RequestSupplier`.
So I thought just checking it inside was okay, but if you think we need to defense there I can add it. 😄

Reply:
In general I'm worried about if we have to be defensive external to the armeria or not, based on line 109. It is scary that we have to check first..

Reply:
Nevermind. this is our own code.. it is still curious. I'll add a comment suggestion..